From: Richard B Winters <rik@mmogp.com>
Date: Fri, 1 May 2015 09:21:51 -0400
Subject: Fix pcsd makefile for gnu linux

 - Patching pcsd makefile with upstream's latest changes, so that 
   the build works properly on debian. Without the patch, the  
   incorrect alias is used for build_gems

Change-Id: Ib30558166f1cc6ee1f075d0b58d08140b21bad60
Signed-off-by: Richard B Winters <rik@mmogp.com>
---
 pcsd/Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/pcsd/Makefile
+++ b/pcsd/Makefile
@@ -1,9 +1,9 @@
 REL_INFO := $(shell grep -q -i "release 6" /etc/redhat-release ; echo $$?)
 
-ifeq (${REL_INFO},1)
-  build_gems: build_gems_normal
-else
+ifeq (${REL_INFO},0)
   build_gems: build_gems_rhel6
+else
+  build_gems: build_gems_normal
 endif
 
 build_gems_normal:
